mzLib/Proteomics/ProteolyticDigestion/PeptideWithSetModifications.cs

```diff
@@ -14,7 +14,9 @@ public class PeptideWithSetModifications : ProteolyticPeptide
     {
         public string FullSequence { get; private set; } //sequence with modifications
         public readonly int NumFixedMods;
-
+        // Parameter to store a hash code corresponding to a Decoy peptide
+        // If 
+        public int? DecoyHash { get; private set; }
```

Comment:
Let's call this PairedTargetDecoyHash because it goes both ways. For a target, its the decoy hash. For the decoy, its the target has. And let's add comments in the code that clearly conveys that information.

Reply:
I already fixed that in the most recent commit.